Microsoft.Toolkit.Uwp/Helpers/StreamHelper.cs

```diff
@@ -184,6 +188,7 @@ public static Task<bool> IsLocalCacheFileExistsAsync(string fileName)
         /// <param name="workingFolder">Folder to use.</param>
         /// <param name="fileName">Relative name of the file to open. Can contains subfolders.</param>
         /// <returns>True if file exists.</returns>
+        [Obsolete("Use StorageFileHelper.FileExistsAsync instead.")]
         public static async Task<bool> IsFileExistsAsync(
             this StorageFolder workingFolder,
             string fileName)
```

Comment:
As IsFileExistsAsync is marked obsolete I think this is already a good reason no to use it